### DISCUSSION TITLE
How do I download a video as an MKV?

### DISCUSSION THREAD

**Discussion by mattm3** [Tzahi12345/YoutubeDL-Material#1029] [Q&A] [answered]
Hi,

I'm using this on Unraid and I would like to download the videos in the MKV format. I have tried adding various options in the custom args section but I haven't been able to get it to work.

Am I doing this wrong? If possible, could someone explain how to do it as if I were a 5 year old as this is all new to me.

Thanks

**Comment by Tzahi12345** [comment, marked as answer]
Not currently possible, we only support mp4 and mp3. We want to get away from that but right now it's just those two file types.

**Reply by mattm3** [reply]
Oh OK thanks for letting me know 🙂

**Reply by Tzahi12345** [reply]
Just found the relevant issue for this if you'd like to track that: https://github.com/Tzahi12345/YoutubeDL-Material/issues/544

**Reply by mattm3** [reply]
Awesome, thanks 😊
